radixdlt-engine/src/main/java/com/radixdlt/engine/parser/REParser.java

```diff
@@ -275,4 +262,23 @@ public ParsedTxn parse(Txn txn) throws TxnParseException {
         payloadHashAndSig,
         parserState.disableResourceAllocAndDestroy);
   }
+
+  private Pair<HashCode, ECDSASignature> calculatePayloadHashAndSign(
```

Comment:
Because `Sign` looks like a verb, I think a slightly clearer name is `calculatePayloadHashAndSignature`

Reply:
Actually, even this name does not reflect the real purpose of this method. Reworked it a bit and fixed naming.